### DISCUSSION TITLE
Class component not called

### DISCUSSION THREAD

**Discussion by ymauron** [nWidart/laravel-modules#1433] [General]
Hi, 

I created a class component within a module but when I call it, it seems that the render function is never called. Here is my setup : 

- I have a module called "Website", which contains Resources/views/components/website-table.blade.php and Resources/views/components/WebsiteTable.php
- My WebsiteServiceProvider.php contains the following registration : 

`Blade::component('Website::website-table', WebsiteTable::class );`

- I call my module the folling way : 

`<x-website::websitetable/>`

This doens't generate any error, but the content of Resources/views/components/website-table.blade.php is not shown and neither the constructor, nor the render method of Resources/views/components/WebsiteTable.php is called. 

Do you have any idea on what I'm doing wrong here ?

**Comment by dcblogdev** [comment]
with blade components you don't need to register them when using just a view but the filename should match the reference.

For example `Resources/views/components/website-table.blade.php` would be loaded as `<x-website::website-table/>`
